src/hotspot/share/gc/z/zAddressSpaceLimit.cpp

```diff
@@ -46,6 +47,6 @@ size_t ZAddressSpaceLimit::mark_stack() {
 
 size_t ZAddressSpaceLimit::heap_view() {
   // Allow all heap views to occupy 50% of the address space
```

Comment:
The "50%" description is inaccurate, right?

Reply:
No, that's still correct.